imap/lmtp_sieve.c

```diff
@@ -191,7 +191,9 @@ static int getmailboxidexists(void *sc, const char *extname)
 {
     script_data_t *sd = (script_data_t *)sc;
     const char *userid = mbname_userid(sd->mbname);
-    char *intname = mboxlist_find_uniqueid(extname, userid, sd->authstate);
```

Comment:
Does jmap_sieve.c need this same change?

Reply:
Also imap/http_applepush.c

Reply:
jmap_sieve.c for sure.  Uncertain on http_applepush.c.  Let's discuss with @brong on Monday

Reply:
Oh did we figure out the applepush questioN?